recipe/meta.yaml

```diff
@@ -9,48 +9,52 @@ source:
   sha256: e383702853236004e5b08e424b8afe9b53fe9f31aaa7a5382f39d9533eb7c043
 
 build:
-  number: 0
-  skip: true  # [win]
+  number: 1
+  # libgpg-error and libgcrypt aren't available on s390x
+  skip: true  # [win or (linux and s390x)]
   run_exports:
     # compatibility is not tracked at abi-laboratory, assume same as libxml2
     - {{ pin_subpackage('libxmlsec1', max_pin='x.x') }}
-  ignore_run_exports:
-    - libiconv         # [not linux]
-    - icu              # [not linux]
-    - libgpg-error     # [not linux]
-    - zlib             # [unix]
-    - xz               # [unix]
+
 requirements:
   build:
     - {{ compiler('c') }}
     - make
     - libtool
   host:
     - icu
-    - libgcrypt
-    - libgpg-error
-    - libiconv  # [not linux]
+    - libgcrypt 1.9.3
+    - libgpg-error 1.42
+    - libiconv 1.16  # [not linux]
     - libtool
```

Comment:
Why is libtool both in `build` and in `host`?

Reply:
One of the files requires it. If I remove it from host the build will fail